myhpi/static/scss/footer.scss

```diff
@@ -1,5 +1,5 @@
 .footer-myhpi {
-    background-color: var(--bs-gray-700);
+    background-color: var(--bs-gray-800);
 
     .page-content {
```

Comment:
The border between top bar and content is not visible any more. For better distinction, could we introduce it again?

This is how it looked:

![grafik](https://github.com/user-attachments/assets/df12b8cf-f916-4371-90e3-50daa14499f3)